web/vue-app/src/App.vue

```diff
@@ -1,28 +1,113 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <div id="more-rent" class="results-gif" v-if="dataReceived && goodResult"></div>
+    <div id="less-rent" class="results-gif" v-if="dataReceived && (goodResult === false)"></div>
+    <results-component v-if="dataReceived" v-bind:inputs="inputs" />
+    <indexation-form   v-else @add:input="sendInputs" />
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
+import IndexationForm from './components/IndexationForm.vue'
+import ResultsComponent from './components/ResultsComponent.vue'
 
 export default {
   name: 'App',
   components: {
-    HelloWorld
+    IndexationForm,
+    ResultsComponent
+  },
+  data() {
+    return {
+      inputs: [
+        {
+          id: 1,
+          start: "01/09/2010",
+          signed: "25/07/2010",
+          region: "brussels",
+          rent: 500,
+          base_index: 1112.74,
+          current_index: 131.72,
+          new_rent: 584.18
+        },
+      ],
+      dataReceived: false,
+      goodResult: false
+    }
+  },
+  methods: {
+    addInput(input) {
+      const lastId =
+        this.inputs.length > 0
+          ? this.inputs[this.inputs.length - 1].id
+          : 0;
+      const id = lastId + 1;
+      const newInput = { ...input, id };
+
+      this.inputs = [...this.inputs, newInput];
+  },
```

Comment:
I was meaning to  get back to this, once I stopped keeping the form inputs and using `v-for` to display the history. I wanted to keep this in though, because I thought it would be nice to show previous searches, but (for some obvious reason I had not found yet) the inputs were split in two in the table with the `v-for`: one line showed the form inputs without the response, while the next line was showing the responses but not the form inputs (because at the time I had not wholly understood where the separation occurred). Indeed, this had no place in the "final" version, cheers!